lib/swiss_village_directory/repository.rb

```diff
@@ -7,7 +7,7 @@ class Repository
 
     def initialize
       filename = "#{File.dirname(__FILE__)}/../../data/PLZO_CSV_WGS84.csv"
-      options = { col_sep: ';', encoding: 'ISO-8859-1:UTF-8', headers: :first_row }
+      options = { col_sep: ';', encoding: 'UTF-8', headers: :first_row }
```

Comment:
The encoding used to be `ISO-8859-1` for words with umlauts. This changed to be just `UTF-8` which is cleaner anyways.